docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -40,6 +41,66 @@
 | [v1.0.1](Data_Portal_Release_Notes.md#release-101) | May 18, 2016 |
 
 ---
+## Release 2.4.0
+
+* __GDC Product__: GDC Data Portal
+* __Release Date__:  March 13, 2025
+
+### New Features and Changes
+* __New CNV Categories__:
+    * The genes table and its associated TSV download in __Mutation Frequency__ have been updated with the new CNV categories. <!--PEAR-2135/2137-->
+    * The Cancer Distribution tool in the __Gene Summary Page__ has been updated to reflect the new CNV categories. <!--PEAR-2136/2291-->
```

Comment:
this will likely change, e.g. may merge all of SJ and our changes into one giant paragraph or something.